cogs/wordfilter/wordfilter.py

```diff
@@ -1,523 +1,523 @@
-"""Word Filter cog.

```

Comment:
Let's get rid of this file from this review (we'll deal with the line endings later):
1. Comment out the following line from .gitattributes:
```
* text eol=lf
```
2. Merge in changes from V3/testing, and resolve conflict by keeping changes from V3/testing for wordfilter.py.